File: src/org/exobel/routerkeygen/AliceMagicInfo.java
package org.exobel.routerkeygen;

import java.io.Serializable;

public class AliceMagicInfo implements Serializable{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1640975633984337261L;
	String alice;
	int [] magic;
	String serial;
	String mac;
	public AliceMagicInfo(String alice,  int[] magic,
			String serial, String mac) {
		this.alice = alice;
		this.magic = magic;
		this.serial = serial;
		this.mac = mac;
	}
}

File: qt/alicemagicinfo.cpp
#include "alicemagicinfo.h"

AliceMagicInfo::AliceMagicInfo(  QString a , int ma[2] , QString s , QString m ) :
        alice(a) , serial(s) , mac(m)
{
    this->magic[0] = ma[0];
    this->magic[1] = ma[1];
}
